Cargo.toml

```diff
@@ -21,6 +21,10 @@ websocket-codec = "0.5"
 tokio-util = { version = "0.7", features = ["codec"] }
 pea2pea = "0.40"
 base64 = "0.13"
+reqwest = { version = "0.11" }
+rmp-serde = "1.0.0"
+data-encoding = "2.3"
+radix_fmt = "1.0"
```

Comment:
A little OCD thing: should we keep deps in alphabetical order?

Reply:
Ooh, okay. Is there an fmt tool for the Cargo files? 😀

Reply:
I don't know, I've always done this manually 🙈

Reply:
there is one, but I forgor the name

Reply:
[Here it is](https://lib.rs/crates/cargo-sort). Cool!
Would be fun if we could run it on every PR, but this is not installed by default.

Done